MultimodalQnA/ui/gradio/requirements.txt

```diff
@@ -1,6 +1,7 @@
-gradio==5.11.0
-gradio_pdf==0.0.19
+gradio==5.17.1
+gradio_pdf==0.0.20
 moviepy==1.0.3
 numpy==1.26.4
 opencv-python==4.10.0.82
 Pillow==10.3.0
+pydantic==2.10.6
```

Comment:
Gradio already brings in pydantic as a dependency, but the latest version of pydantic release today (3/27/2025) appears to be problematic where the UI won't even load. Pegging to the previous release of pydantic fixes the issue.